package.json

```diff
@@ -21,27 +34,12 @@
     "coverage-publish": "aegir coverage publish",
     "bench": "node benchmarks/hash.js"
   },
-  "pre-push": [
-    "lint",
-    "test"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/multiformats/js-multihashing-async.git"
-  },
-  "keywords": [
-    "multihash"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/multiformats/js-multihashing-async/issues"
-  },
+  "repository": "github:multiformats/js-multihashing-async",
```

Comment:
Somewhere (I can try to find it), there was some consensus on using `lint` as pre-push hook (I personally also use `test`, but that's up to every maintainer).